documentation/release-process.md

```diff
@@ -127,4 +133,12 @@ The release image is `mcr.microsoft.com/dotnet/monitor`. The tag list is https:/
 1. Update the `AutoUpdateDockerBranches` variable to `refs/heads/main` in the [internal pipeline](https://dev.azure.com/dnceng/internal/_build?definitionId=954) to begin the consumption of nightly builds into dotnet-docker. Note this should not necessarily be done right after the release, but after the merge from main to nightly in the dotnet-docker repo (such as https://github.com/dotnet/dotnet-docker/pull/4741). Include additional branch references and semi-colon delimit each value e.g. `refs/heads/main;refs/heads/feature/9.x`.
 1. Review and merge the automatically create `Register new release information` PR.
 1. For each release, push its corresponding tag to the `shipped/v<version>` branch in the [internal repository](https://dev.azure.com/dnceng/internal/_git/dotnet-dotnet-monitor) e.g `v8.0.0-rc.1.23458.6 -> shipped/v8.0`. If done correctly, this should be a fast-forward merge.
+```
+git remote add public https://github.com/dotnet/dotnet-monitor.git
```

Comment:
I think these instructions should have the release champ checkout the tag of the release and push it to the corresponding `shipped/v*` branch. This would eliminate the possibility of creating merge commits on the local machine and would show if the history had changed for some reason; this push would be a fast-forward and any other type of merge would indicate a discrepancy.